src/main/java/at/aau/se2/handler/game/subhandlers/RequestUsernamesHandler.java

```diff
@@ -25,13 +28,18 @@ public void handleMessage(WebSocketSession session, JsonNode msg, Lobby lobby) {
     public String convertToJson() {
         StringBuilder builder = new StringBuilder();
         builder.append("{ 'type': 'REQUEST_USERNAMES', 'usernames': [");
-
         for(String u : GameHandler.getUsernames()){
             if(GameHandler.getUsernames().get(GameHandler.getUsernames().size()-1).equals(u))
-                  builder.append("'").append(u).append("']}");
+                builder.append("'").append(u).append("']}");
             else
                 builder.append("'").append(u).append("',");
         }
         return builder.toString();
     }
+    public void updateOtherUser(Lobby lobby) throws IOException {
```

Comment:
Sendet die Message an alle User, inklusive dem Aufruf von davor
**2 Lösungsansätze:**
1. sendMessage Call in Line 19 entfernen
2. Abfrage: `if(a.getSession() != session)`in die Methode einbauen